Makefile

```diff
@@ -0,0 +1,21 @@
+deploy: build
+	@git describe --tags > ./appversion
+	@heroku docker:release --app origami-registry-eu
+	@rm -f ./appversion
+
+build:
+	@if [[ "$$(docker-machine ls | grep dev)" == *"Stopped"* ]]; \
```

Comment:
Could this be a one-line conditional that runs `exit 0` if the machine is not stopped? Then you wouldn't need all the escaped line breaks